casper/src/test/scala/coop/rchain/casper/util/rholang/MergingBranchMergerSpec.scala

```diff
@@ -29,7 +29,7 @@ import scala.collection.Seq
 
 class MergingBranchMergerSpec extends FlatSpec with Matchers {
 
-  val genesisContext             = GenesisBuilder.buildGenesis(validatorsNum = 50)
+  val genesisContext             = GenesisBuilder.buildGenesis(validatorsNum = 5)
```

Comment:
This change is to mitigate issue <https://github.com/rchain/rchain/issues/3349> of false positive conflict detection. Once it is closed, num of validators should be increased